### PR TITLE
1.0.0 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Navigation buttons at the bottom of each .md file
- Fixed Typo, Grammar, and information flow issues

### CODE REVIEW

**Review by cs-rohit-khune — APPROVED** [review]
LGTM

File: docs/contents.md [hunk 1]
```diff
@@ -5,33 +5,39 @@
...
+|:-----------------------------------|
+
+| Playbook Name                                   | Description                                                                                                                                                                |
+|:------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Investigate Brute Force Attempt                 | Investigates login failures and also identifies other impacted assets that have been victims of the brute force attempts from a particular source of attack                |
+| Investigate Brute Force Attempt (FortiSIEM)     | Investigates login failures from FortiSIEM and also identifies other impacted assets that have been victims of the brute force attempts from a particular source of attack |
+| Scenario - FortiSIEM (Brute Force Attack) Alert | Generates a demo alert for Brute Force Attack                                                                                                                              |
```
Comment (cs-abhinav-goel) [inline docs/contents.md:38]: In this we should add the tag "Renamed"

File: docs/setup.md [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
 ## Prerequisites
 
-|Solution Pack Name|Purpose|
-| :- | :- |
-|SOAR Framework 1.0.0|Required for Incident Response modules|
-|SOC Simulator 1.0.1|Required for Scenario Module and SOC Simulator connector|
+| Solution Pack Name | Version | Purpose                                                  |
+|:-------------------|:-------:|:---------------------------------------------------------|
+| SOAR Framework     | v1.0.0  | Required for Incident Response modules                   |
+| SOC Simulator      | v1.0.1  | Required for Scenario Module and SOC Simulator connector |
```
Comment (cs-abhinav-goel) [inline docs/setup.md]: SOC simulator version >> 1.0.2
SFSP >> 2.1.0
@cs-rohit-khune @cs-abhishek-shukla : Please confirm.

File: docs/contents.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md:38]: Redundant. As we are mentioning the same in release notes.

File: docs/setup.md (hunk 2 above)
Comment (cs-rohit-khune) [reply docs/setup.md]: The SP will work fine on lower versions of SFSP and SOC SP.  We haven't made any version-dependent changes.

File: docs/setup.md [hunk 3]
```diff
@@ -10,10 +10,10 @@
 
 ## Prerequisites
 
-|Solution Pack Name|Purpose|
-| :- | :- |
-|SOAR Framework 1.0.0|Required for Incident Response modules|
-|SOC Simulator 1.0.1|Required for Scenario Module and SOC Simulator connector|
+| Solution Pack Name | Version | Purpose                                                  |
+|:-------------------|:-------:|:---------------------------------------------------------|
+| SOAR Framework     | v1.0.0  | Required for Incident Response modules                   |
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: or later

File: docs/setup.md (hunk 2 above)
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: 1.0.1 or later - @cs-abhishek-shukla

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**